spec/controllers/physical_rack_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe PhysicalRackController do
+  render_views
+
+  let!(:server) { EvmSpecHelper.local_miq_server(:zone => zone) }
+  let(:zone) { FactoryGirl.build(:zone) }
+
+  before(:each) do
+    stub_user(:features => :all)
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user)
+    ems = FactoryGirl.create(:ems_physical_infra)
+    @physical_rack = FactoryGirl.create(:physical_rack,
+                                        :ems_id => ems.id,
+                                        :id     => 1)
+  end
+
+  describe "#show_list" do
+    subject { get :show_list }
+
+    it do
+      is_expected.to have_http_status 200
+      is_expected.to render_template(:partial => "layouts/_gtl")
+    end
+  end
+
+  describe "#show" do
+    context "with valid id" do
+      subject { get :show, :params => {:id => @physical_rack.id} }
+
+      it "should respond to show" do
+        is_expected.to have_http_status 200
+        is_expected.to render_template(:partial => "layouts/_textual_groups_generic")
+      end
+    end
+    context "with invalid id" do
+      subject { get :show, :params => {:id => 2 } }
```

Comment:
Please align the curly braces consistently and use parentheses for arguments to improve readability.